src/Shared/Recruit.Vacancies.Client/Recruit.Vacancies.Client.csproj

```diff
@@ -37,7 +37,7 @@
     <PackageReference Include="RestSharp" Version="106.6.10" />
     <PackageReference Include="SFA.DAS.Account.Api.Client" Version="1.6.1601" />
     <PackageReference Include="SFA.DAS.Http" Version="1.2.4" />
-    <PackageReference Include="SFA.DAS.VacancyServices.Wage" Version="1.0.8" />
+    <PackageReference Include="SFA.DAS.VacancyServices.Wage" Version="1.0.13" />
```

Comment:
why is the wage updated?

Reply:
The updated VacacnyServices.Wage package was to change the content that is returned from the GetYearlyMinimumWageText method

Reply:
👍